src/drive/targets/browser/index.jsx

```diff
@@ -61,11 +61,23 @@ document.addEventListener('DOMContentLoaded', () => {
   const store = configureStore(client, polyglot.t.bind(polyglot))
 
   render(
-    <I18n lang={data.cozyLocale} polyglot={polyglot}>
-      <CozyProvider store={store} client={client}>
-        <Router history={history} routes={AppRoute} />
-      </CozyProvider>
-    </I18n>,
+    <HotedApp
+      lang={data.cozyLocale}
+      polyglot={polyglot}
+      store={store}
+      client={client}
+      history={history}
+    />,
     root
   )
 })
+
+const AppComponent = props => (
+  <I18n lang={props.lang} polyglot={props.polyglot}>
+    <CozyProvider store={props.store} client={props.client}>
+      <Router history={props.history} routes={AppRoute} />
+    </CozyProvider>
+  </I18n>
+)
+
+const HotedApp = hot(module)(AppComponent)
```

Comment:
What's `module` here?

Reply:
This is the "old" API. But compatible with parcel. And it seems to work well. I didn't try the new one (without `module`). We could try for a new entry. It seems to be more resilient on JS errors... 

`module` is the global `webpack` object. https://webpack.js.org/api/module-variables/